packages/babel-plugin/src/utils/build-styled-component.ts

```diff
@@ -101,6 +104,36 @@ const traverseStyledBinaryExpression = (node: t.BinaryExpression, nestedVisitor:
   return node;
 };
 
+/**
+ * Traverses a tagged template expression looking for any arrow functions,
```

Comment:
hmm this only handles tagged template expression? Does that mean we have do it again for Object styles? :(

Reply:
Good point! Let me check